.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,50 @@
+name: Publish to PyPi
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.11"
+    - name: Install pypa/build
+      run: >-
+        python3 -m
+        pip install
+        build
+        --user
```

Comment:
Is this `--user` needed?

Reply:
I copied this from PyPi's website. I'll remove `--user` in any case.

Reply:
Resolved by [fc216b8](https://github.com/poliastro/czml3/pull/138/commits/fc216b83ed7c0e9fb35df1ec3d2bb589be8886a2)